### ISSUE TITLE
Spam in logs - "enabled capabilities for version"

### ISSUE THREAD

**Issue by sheerun** [kubernetes/minikube#223]
I'm on OSX, 10.10.5. I use minikube 0.3.0, and virtualbox.

After I run "minikube logs", I see following ever growing spam:

```
2016-06-25 12:49:16.213069 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:16.247722 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:16.714497 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:16.749475 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:17.215195 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:17.250204 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:17.716191 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:17.750414 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:18.216368 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:18.251182 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:18.717820 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:18.752310 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:19.219113 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:19.252781 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:19.720139 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:19.754217 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:20.220725 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:20.256163 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:20.721921 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:20.756733 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:21.223302 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:21.257513 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:21.724626 I | v2http: enabled capabilities for version 2.3.0
2016-06-25 12:49:21.762113 I | v2http: enabled capabilities for version 2.3.0
```

**Comment by dlorenc** [comment]
Thanks for catching this. We'll see if we can quiet this down.

**Comment by MikeCoutts** [comment]
FYI I'm on 0.4.0 and now getting the following from minikube logs

2016-06-30 00:27:31.839122 I | v2http: enabled capabilities for version 2.3.0
I0630 00:27:32.220768    2783 disk_manager.go:107] Running out of space on disk for "root": available 201 MB, threshold 256 MB
2016-06-30 00:27:32.339859 I | v2http: enabled capabilities for version 2.3.0

2016-06-30 04:39:32.298576 I | v2http: enabled capabilities for version 2.3.0
I0630 04:39:32.340002    2783 disk_manager.go:107] Running out of space on disk for "root": available 200 MB, threshold 256 MB

So losing 1MB of space every 4 hours.

**Comment by dlorenc** [comment]
Yeah this is definitely annoying. We'll have a fix in 0.5, and try to get that out Friday or Monday.
